code/game/objects/items/stacks/stack_recipes.dm

```diff
@@ -340,6 +340,7 @@ var/list/datum/stack_recipe/metal_recipes = list (
 		), 2),
 	new/datum/stack_recipe("table parts", /obj/item/weapon/table_parts,                           2                                ),
 	new/datum/stack_recipe("rack parts",  /obj/item/weapon/rack_parts,                                                             ),
+	new/datum/stack_recipe("crate shelf parts", /obj/item/weapon/rack_parts/shelf,                3                                ),
```

Comment:
Looking at the sprite, clearly, this rack needs to cost 4 rods in addition to the 3 metal sheet requirement.........................

Reply:
Increased to 5 sheets